src/components/box/box.component.js

```diff
@@ -9,6 +10,38 @@ const Box = styled.div`
   ${layout}
   ${flexbox}
   ${color}
+
+  ${({ scrollVariant, theme }) => css`
+    &::-webkit-scrollbar {
```

Comment:
As my comment below, do we not just want this added when the `scrollVariant` prop has been added?

Reply:
When we make this change, would this be useful to move into a helper, like I did for `${color}`, that would allow other components to add it without extending `Box`? Or should we do this at a later date?

Reply:
Moved this to only be added when scrollVariant isnt undefined